projects/real_world_ovmm/evalai_env.py

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env python
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+from datetime import datetime
+from typing import Optional, Tuple
+
+import click
+import rospy
+
+from home_robot.agent.ovmm_agent.ovmm_agent import OpenVocabManipAgent
+from home_robot.motion.stretch import STRETCH_HOME_Q
+from home_robot_hw.env.stretch_pick_and_place_env import StretchPickandPlaceEnv
+from home_robot_hw.utils.config import load_config
+
+import grpc
+import evaluation_pb2
+import evaluation_pb2_grpc
+
+def grpc_dumps(entity):
+    return pickle.dumps(entity)
+
+def grpc_loads(entity):
+    return pickle.loads(entity)
+
+def metric_show_participant(phase):
+    if phase == "1734" or phase == "1731":
+        return False
+    return True
+
+
+def update_submission_result_and_exit(
```

Comment:
I don't think we will have the ability to upload results automatically to the evalai leaderboard so this could be deleted.

Reply:
Oh ok. I wasn't sure how the real world results would interact with eval-ai

Reply:
I don't think we need to submit them to eval-ai at all do we?

Reply:
They aren't part of the leaderboard results?